### PR TITLE
Show license footer only when appropriate

### PR DESCRIPTION
**Pull request by raboof**
Requires https://github.com/lightbend/paradox/pull/343

### CODE REVIEW

File: theme/src/main/assets/page.st [hunk 1]
```diff
@@ -121,8 +121,10 @@ $source()$
 
 <footer class="page-footer row clearfix">
 <img class="akka-icon float-left show-for-medium" src="$page.base$images/akka-icon.svg"/>
+$if(page.property_is.("project.license").("Apache-2.0"))$
```
Comment (ennru) [inline theme/src/main/assets/page.st]: Do all of our projects format the identifier the same?

Comment (raboof) [reply theme/src/main/assets/page.st]: Perhaps not, but in that case we should fix it there as this is the identifier as specified in https://spdx.org/licenses/

Comment (ennru) [reply theme/src/main/assets/page.st]: Maybe move the whole `copyright` section into an include file so that we can override it where appropriate.

Comment (raboof) [reply theme/src/main/assets/page.st]: (might be neat but perhaps for another PR?)

**Comment by ennru** [comment]
Needs a rebase.

**Comment by raboof** [comment]
rebased, needs a release of paradox 0.6.3

**Comment by raboof** [comment]
released paradox 0.6.3...

**Pull request merged**